PAS-84

seiyaterada opened the pull request.

seiyaterada committed:
Initial commit

seiyaterada committed:
Initial commit

seiyaterada committed:
Update updateSemester

seiyaterada committed:
Update updateSemester

seiyaterada committed:
Changed AddCourseButton to function

seiyaterada commented:
Just waiting on PAS-102 in order to finish the ticket! #69

seiyaterada committed:
Final commit before ready for PR. Add Course button now works

BraidenCutforth commented:
@seiyaterada Let's rebase this branch onto the tip of master. There are a lot of changes showing that you didn't do, making it hard to read. I did a test PR with branch `PAS-84-rebase` if you want to check out what I think the result should be: #80

seiyaterada committed:
Changed some api data types
made some api data types optional since they are undefined until created in
database

Samantha-norrie reviewed (approved):
🔘
